tests/async_client_test.py

```diff
@@ -1805,6 +1805,31 @@ async def test_devices(self, async_client, aioresponse):
         resp = await async_client.delete_devices(devices)
         assert isinstance(resp, DeleteDevicesResponse)
 
+    async def test_update_device(self, async_client: AsyncClient, aioresponse: aioresponses):
+        """Test that we can update a device
```

Comment:
I love to see docstrings in tests 🙏

Reply:
To be fair i took your PR #131 as reference :)